pages/getting-started/installation.mdx

```diff
@@ -17,23 +32,50 @@ For those looking to delve deeper or self-host Snaplet, try our user-friendly `s
   </Tab>
   <Tab>
   ```bash >_&nbsp;terminal
-  # Set up Snaplet with pnpm
-  pnpm dlx snaplet setup
+  # Set up Snaplet as a static binary
+  curl -sL https://app.snaplet.dev/get-cli/ | bash
+  snaplet setup
   ```
   </Tab>
+</Tabs>
+
+
+When you run `snaplet setup`, you will be asked a few questions to complete setup, and generate a [`snaplet.config.ts`](/reference/configuration) configuration file for your project.
+
+## Manual setup
+
+### Installing dependencies
+
+Snaplet requires to following to be present in your project:
+* a `package.json`: to determine where to install snaplet packages to
+* a package manager lockfile to be present for your project - to detect which package manager you are using
```

Comment:
Is it something the user should know? It seems like an implementation detail on our end.

Reply:
Ya, I know what you mean.

Thing is, we skip this step in `snaplet setup` if these conditions aren't met, and send them to these docs. So we need a way to say "here's how you fix those things"